ledger/ledger-api-test-tool-on-canton/BUILD.bazel

```diff
@@ -82,6 +82,7 @@ conformance_test(
         ",ParticipantPruningIT" +  # see "conformance-test-participant-pruning" below
         ",ConfigManagementServiceIT,LedgerConfigurationServiceIT" +  # dynamic config management not supported by Canton
         ",ClosedWorldIT" +  # Canton currently fails this test with a different error (missing namespace in "unallocated" party id)
+        ",CommandServiceIT:CSCreateAndBadExerciseChoice,CommandSubmissionCompletionIT:CSCRefuseBadChoice" +  # FIXME: canton should be updated with last SDK to produce proper output message for those tests to pass
```

Comment:
@oliverse-da Fyi, I inactivated 2 tests. This is due to a mismatch of the error messages because an change in the engine.

Reply:
You should reactivate once you update canton with the next SDK.